cctp_primitives/src/commitment_tree/sidechain_tree_alive.rs

```diff
@@ -85,6 +85,30 @@ impl SidechainTreeAlive {
         self.cert_mt.get_leaves().to_vec()
     }
 
+    // Gets merkle path to the Forward Transfer in the tree
+    pub fn get_fwt_merkle_path(&mut self, leaf_index: usize) -> Option<GingerMHTPath> {
+        match self.fwt_mt.borrow_mut().finalize() {
```

Comment:
It doesn't matter, but there is no need to borrow mut, since finalize accepts an immutable reference

Reply:
Thanks! Done.